book/docs/classes/OpusPool/Methods/buildWithdrawUnstakedTransaction.md

```diff
@@ -0,0 +1,29 @@
+### buildWithdrawUnstakedTransaction
+
+▸ **buildWithdrawUnstakedTransaction**(`params`): `Promise`\<[`UnstakeQueueTransactionData`](../../../interfaces/UnstakeQueueTransactionData.md)\>
+
+Generates transaction data to withdraw from the unstake queue
```

Comment:
Here as well, I think it's worth expanding a bit, maybe explaining that this tx needs to be preceded by the `unstake` one.